sites/platform/config/_default/params.yaml

```
@@ -107,14 +107,14 @@ nonDocsSections:
 # Metadata
 meta:
   sitename: "Platform.sh User Documentation"
-  description: Platform.sh is a second-generation Platform-as-a-Service built especially for continuous deployment. It allows you to host web applications on the cloud while making your development and testing workflows more productive.
+  description: Platform.sh is a second-generation Cloud Application Platform built especially for continuous deployment. It allows you to host web applications on the cloud while making your development and testing workflows more productive.
```

```suggestion
  description: Platform.sh is a Cloud Application Platform built especially for continuous deployment. It allows you to host web applications on the cloud while making your development and testing workflows more productive.
```

I would remove "second-generation" as I don't think it's useful (I'm not even sure what that means 😂 ).